extensions/amp-analytics/0.1/transport.js

```diff
@@ -19,42 +19,55 @@ import {
   parseUrl,
   checkCorsUrl,
 } from '../../../src/url';
+import {createElementWithAttributes} from '../../../src/dom';
 import {dev, user} from '../../../src/log';
 import {loadPromise} from '../../../src/event-helper';
 import {timerFor} from '../../../src/services';
 import {removeElement} from '../../../src/dom';
-import {setStyle} from '../../../src/style';
+import {setStyle, setStyles} from '../../../src/style';
+import {hasOwn, map} from '../../../src/utils/object';
+import {IframeMessagingClient} from '../../../3p/iframe-messaging-client';
 
 /** @const {string} */
 const TAG_ = 'amp-analytics.Transport';
 
-/**
- * @param {!Window} win
- * @param {string} request
- * @param {!Object<string, string>} transportOptions
- */
-export function sendRequest(win, request, transportOptions) {
```

Comment:
Not deleted, just moved inside the Transport class below, because needs to maintain extraData as non-static.